source/_integrations/knx.markdown

```diff
@@ -697,6 +697,11 @@ controller_modes:
   description: Overrides the supported controller modes. Provide the supported `hvac_mode` values for your device.
   required: false
   type: list
+default_controller_mode:
+  description: Overrides the default controller mode. Any Home Assistant `hvac_mode` can be configured. This can for examplt be set to "cool" for cooling only devices.
```

Comment:
```suggestion
  description: Overrides the default controller mode. Any Home Assistant `hvac_mode` can be configured. This can, for example, be set to "cool" for cooling-only devices.
```

Reply:
Thanks 👍